examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
you file should be called plot_elekta_average.py otherwise will not be executed by doc build.

it should plot something to have a thumbnail in the gallery